docs/src/main/sphinx/connector/jdbc-procedures.fragment

```diff
@@ -9,3 +9,16 @@
   USE example.example_schema;
   CALL system.flush_metadata_cache();
   ```
+
+- `system.update('query')`
+
+  The `update` procedure allows you to query the underlying database directly.
+  This can be useful for accessing native features which are not available in Trino
+  or for executing DDL or DML that `query` pass-through table function doesn't support.
```

Comment:
We should maybe pull some info from the javadoc about potentially returning nothing. And instead of DDL and DML talk about insert, update, delete and others.